### PR TITLE
LPS-125868 upgrade schema version

### PR DESCRIPTION
**Pull request by AliciaGarciaGarcia**

### CODE REVIEW

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/DDMServiceUpgrade.java [hunk 1]
```diff
@@ -394,6 +395,10 @@ public void register(Registry registry) {
 			"4.1.0", "4.2.0",
 			new com.liferay.dynamic.data.mapping.internal.upgrade.v4_2_0.
 				UpgradeDDMFormInstanceRecord());
+
+		registry.register(
+			"4.2.0", "4.3.0",
+			new UpgradeDDMFieldAttributeFromDLFileEntryType());
```
Comment (adolfopa) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/DDMServiceUpgrade.java]: Better rename to UpgradeDLFileEntryTypeDDMFieldAttribute

**Review by adolfopa — CHANGES_REQUESTED** [review]
Why cannot this be moved to document-library-service? Is there some dependency?

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/DDMServiceUpgrade.java (hunk 1 above)
Comment (AliciaGarciaGarcia) [reply modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/upgrade/DDMServiceUpgrade.java]: thanks!

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:sf

**Comment by AliciaGarciaGarcia** [comment]
ci:test:relevant

**Comment by AliciaGarciaGarcia** [comment]
https://github.com/liferay-data-engine/liferay-portal/pull/148

**Pull request closed without merging**